docs/pages/access-controls/device-trust/guide.mdx

```diff
@@ -1,462 +1,212 @@
-role 'device-admin' has been created
-```
+### Prerequisites
 
-<Admonition type="tip" title="Editor Role">
-For clusters created using Teleport v12 or newer, the builtin role `editor` has
-the necessary permissions to manage devices. For production setups a separate
-role is still recommended.
-</Admonition>
+(!docs/pages/includes/no-oss-prereqs-tabs.mdx!)
 
-Note that in addition to the usual CRUD verbs (create, read, list, update and
-delete), we have also included `create_enroll_token` and `enroll`. The
-`create_enroll_token` verb is necessary to execute the `tctl devices enroll`
-command; `enroll` is necessary to execute `tsh device enroll`. Both are
-demonstrated in the following section.
+(!docs/pages/includes/device-trust/prereqs.mdx!)
 
-Make sure your user has the `device-admin` role:
+- User with `editor` role.
+  - ```code
+    $ tsh status
+    > Profile URL:      https://teleport.example.com:443
+    Logged in as:       myuser
+    Cluster:            teleport.example.com
+    Roles:              access, auditor, editor
+    Logins:             root, ubuntu, ec2-user
+    Kubernetes:         disabled
+    Valid until:        2023-08-22 03:30:24 -0400 EDT [valid for 11h52m0s]
+    Extensions:         login-ip, permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+    ```
+- Access to a linux server (You can pick any test linux server enrolled in Teleport which
+  your Teleport user is allowed for SSH access).
+  - ```code
+    $ tsh ls
+    Node Name        Address        Labels
+    ---------------- -------------- --------------------------------------
+    ip-172-31-35-170 ⟵ Tunnel
+
+    # test connection to ip-172-31-35-170
+    $ tsh ssh root@ip-172-31-35-170
+    root@ip-172-31-35-170:~#
+    ```
 
-(!/docs/pages/includes/add-role-to-user.mdx role="device-admin"!)
+Once the above prerequisites are met, begin with the following step.
 
-Relogin to fetch updated certificates:
+## Step 1/2. Update user profile to enforce Device Trust
 
-```code
-$ tsh logout; tsh login --proxy=teleport.example.com --user=alice
-```
+To enforce Device Trust, a user must be assigned with a role with Device Trust mode "required".
 
-Before you can enroll the device, you need to register it. To register
-a device, you first need to determine its serial number.
+For this guide, we will use the preset `require-trusted-device` role to update current user profile.
+<Admonition type="warning" title="v13.3.5 and Below">
+  The preset `require-trusted-device` role, as referenced in this guide, is only available
+  from Teleport version 13.3.6 and above. For older Teleport cluster, you will need to update
+  a role with `device_trust_mode: required`.
 
-<Tabs>
-  <TabItem label="macOS">
-    The serial number of a macOS device can be determined in two ways.
+  <Details title="Update a role to enforce Device Trust" closed>
+    For simplicity, the example below updates the preset `access` role but you can update
```

Comment:
> Updating access could have wide-ranging effects.

For pro (already using Teleport) users , yes and I think that group probably understand the implication and will go about updating/creating another role anyway. But users just starting with Teleport, I don't think it will have that serious impact and guiding to create another role just adds fricition while getting started. 

If you think the current copy does not put much emphasis on the risk, I had the following copy before:
"For simplicity, this guide works on the default preset `access` role. If other users in
your cluster are already assigned with this `access` role to access resources, and you do
not want to enforce device trust for them yet, create a separate `device-enforcment`
role and use that role for testing to avoid disruption."

Let me know what you think we should at least use the above copy (or similar variation). But I don't think its a good idea to tell users to create another role for day 1 usage.

Reply:
I like your suggested copy, but feel free to do what you think is best.

Reply:
I'll keep this as is and raise my hand if angry user shows up in the future complaining this guide blew off their team's access.